pkgs/by-name/ni/nix-prefetch-bzr/nix-prefetch-bzr.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
```

Comment:
The prior `#!/bin/sh` was correct as this script is already POSIX-compliant. It doesn’t require the overhead of Bash. See: #462164.